data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py

```diff
@@ -0,0 +1,226 @@
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+from data_pipeline.etl.base import ExtractTransformLoad, ValidGeoLevel
+from data_pipeline.etl.sources.geo_utils import (
+    add_tracts_for_geometries,
+    get_tribal_geojson,
+    get_tract_geojson,
+)
+from data_pipeline.score import field_names
+from data_pipeline.utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class TribalOverlapETL(ExtractTransformLoad):
+    """Calculates the overlap between Census tracts and Tribal boundaries."""
+
+    # Metadata for the baseclass
+    NAME = "tribal_overlap"
+    GEO_LEVEL = ValidGeoLevel.CENSUS_TRACT
+
+    PUERTO_RICO_EXPECTED_IN_DATA = False
+    ALASKA_AND_HAWAII_EXPECTED_IN_DATA = True
+    EXPECTED_MISSING_STATES = [
+        "10",
+        "11",
+        "13",
+        "15",
+        "17",
+        "18",
+        "21",
+        "24",
+        "33",
+        "34",
+        "39",
+        "50",
+        "51",
+        "54",
```

Comment:
Can we document why this is?

Reply:
Lol I honestly have no idea either, so I'll give that to @lucasmbrown-usds --- my best guess is those states have no tribal lands in them.

Reply:
Yes exactly. I'll add a comment though, awesome point @esfoobar-usds that we should be commenting these.